qutebrowser/browser/browsertab.py

```diff
@@ -196,6 +196,10 @@ def next_result(self, *, result_cb=None):
         """
         raise NotImplementedError
 
+    def searching(self):
+        """Return True if we are currently searching or not."""
+        raise NotImplementedError
+
```

Comment:
I'm also not sure about the name - what I need for #2442 is something set to true while Chromium is searching (i.e. between `findText` and the callback being called), while your `searching` means "there are some highlighted search results on the screen". I'm not sure if there's a better name though...